src/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/SdkTemplateTests.cs

```diff
@@ -72,6 +72,112 @@ public void VerifyMSTestTemplate(DotNetLanguage language)
             DotNetSdkActions.Test);
         newTest.Execute(_sdkHelper, _scenarioTestInput.TestRoot);
     }
+    
+    [Theory]
+    [InlineData(DotNetLanguage.CSharp)]
+    public void VerifyCSharpWPFTemplate(DotNetLanguage language)
+    {
+        var newTest = new SdkTemplateTest(
+            nameof(SdkTemplateTest), language, _scenarioTestInput.TargetRid, DotNetSdkTemplate.Wpf,
+            DotNetSdkActions.Test | DotNetSdkActions.Publish);
+        newTest.Execute(_sdkHelper, _scenarioTestInput.TestRoot);
+    }
+    
+    [Theory]
+    [InlineData(DotNetLanguage.VB)]
```

Comment:
What is the reasoning in declaring this as a separate test from the CSharp equivalent versus placing multiple data attributes on a shared testcase?  Furthermore the majority of these test cases could the implemented as a single test see the pattern used in the [source-build smoke tests](https://github.com/dotnet/dotnet/blob/main/test/Microsoft.DotNet.SourceBuild.SmokeTests/BasicScenarioTests.cs#L28).  This makes maintenance nicer IMO.  It also makes it easier to ensure the tests are identical in scenarios like testing different languages.

Reply:
The first issue here has been fixed. The second, while it does sound like a good idea is a bit out of scope for this PR. I may explore that going forward.

Reply:
I logged https://github.com/dotnet/scenario-tests/issues/22 for the second.